### PR TITLE
Added Adcrowd template

### PR DESCRIPTION
**Pull request by martijnvv**

### CODE REVIEW

File: tags/Adcrowd/Adcrowd.tpl [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+// Enter your template code here.
+//var log = require('logToConsole');
+const injectScript = require('injectScript');
+
+var _ac = {};
```
Comment (sahava) [inline tags/Adcrowd/Adcrowd.tpl]: Should be `const _ac = {};`

File: tags/Adcrowd/Adcrowd.tpl [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+if(data.orderValue !== undefined){
+	_ac.value = data.orderValue;
+}
+if(data.productId !== undefined){
+	_ac.product = data.productId;
+}
+if(data.orderId !== undefined){
+	_ac.order_id = data.orderId;
+}
+
```
Comment (sahava) [inline tags/Adcrowd/Adcrowd.tpl:166]: Would probably be more legible if you refactor it to:

 ```
const _ac = {
  value: data.orderValue,
  product: data.productId,
  order_id: data.orderId
};
```

This should work fine unless there's some specific checking to see if fields contain undefined values.

File: tags/Adcrowd/Adcrowd.tpl [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+var trackingUrl = "https://pixel.adcrowd.com/smartpixel/" + data.pixelId + ".js";
```
Comment (sahava) [inline tags/Adcrowd/Adcrowd.tpl]: Should be 

`const trackingUrl = 'https://pixel.adcrowd.com/smartpixel/' + data.pixelId + '.js';`

File: tags/Adcrowd/Adcrowd.tpl [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+injectScript(trackingUrl, data.gtmOnSuccess,data.gtmOnFailure);
+
+//log('script =', trackingUrl);
```
Comment (sahava) [inline tags/Adcrowd/Adcrowd.tpl:170]: I don't see you using the `_ac` local variable anywhere, so the initialization of the variable doesn't actually do anything.

What's the purpose of `_ac`? Are you supposed to set it as a global object or something?

File: tags/pardot/Pardot.tpl [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+//var log = require('logToConsole');
+const injectScript = require('injectScript');
+const setInWindow = require('setInWindow');
+
+setInWindow('piAId', data.piAId);
+setInWindow('piCId', data.piCId);
+
+var trackingUrl = "https://pi.pardot.com/pd.js";
```
Comment (sahava) [inline tags/pardot/Pardot.tpl]: Should be `const trackingUrl = 'https://pi.pardot.com/pd.js';`

File: tags/pardot/Pardot.tpl [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+//var log = require('logToConsole');
+const injectScript = require('injectScript');
+const setInWindow = require('setInWindow');
+
+setInWindow('piAId', data.piAId);
+setInWindow('piCId', data.piCId);
+
```
Comment (sahava) [inline tags/pardot/Pardot.tpl:193]: If you want to overwrite existing `piAid` and `piCid`, you can set a third parameter to `true` in the `setInWindow`, like:

```
setInWindow('piAid', data.piAid, true);
```

File: tags/Adcrowd/Adcrowd.tpl (hunk 2 above)
Comment (martijnvv) [reply tags/Adcrowd/Adcrowd.tpl:166]: That's correct. In some cases, fields may be undefined.

Comment (sahava) [reply tags/Adcrowd/Adcrowd.tpl:166]: Yeah but setting them in the object even if they're undefined should be OK as the JS library should be structured in a way to automatically exclude undefined items (e.g. how Facebook, GA works). Naturally, if this is not the case and `undefined` is significant, then the original way of conditionally populating the object is better :)

File: tags/Adcrowd/README.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# Adcrowd Google Tag Manager Template
+This is the template for Adcrowd Smartpixels in Google Tag Manager
+
```
Comment (sahava) [inline tags/Adcrowd/README.md:13]: One last review request - could you update the README for Adcrowd to match the format used by other tags (same as how you did Pardot)?

**Review by sahava — CHANGES_REQUESTED** [review]

**Pull request merged**